test/includes.spec.js

```diff
@@ -13,30 +13,36 @@ describe("includes", function () {
     function setupLinks(_ids) {
         ids = _ids;
 
-        function link(url, path, value) {
+        function link(resource, resourceId, link, linkId) {
+          const url = `/${resource}/${resourceId}`;
```

Comment:
Could you explain this change? It seems you followed an TODO comment that led you to this change.